canonical/ubuntu.com
# WD-10435 - intel iot page rebrand

## Done

Rebrand intel-iot page.

## QA

- [demo link](https://ubuntu-com-13970.demos.haus/download/iot/intel-iot)
- [copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit)
- [figma](https://www.figma.com/design/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=1702-5346&m=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc and figma

## Issue / Card
[WD-10435](https://warthogs.atlassian.net/browse/WD-10435)


[WD-10435]: https://warthogs.atlassian.net/browse/WD-10435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
